CONTRIBUTING.md

```diff
@@ -24,15 +24,16 @@ The submitted code should adhere to the following:
 
 ### Do's
 
-* For every pull request, make sure all commits are consolidated into one commit before
-  sending the pull request.
+* Keep pull requests focused on addressing one issue or feature. If multiple separate issues
```

Comment:
[Based on what happened today](https://github.com/google/EarlGrey/commit/f2cc0c464cc99d280bf75561ba44b844e83e382c), this info has to change.

>  If multiple separate issues are fixed by a pull request, prefer committing the fixes under multiple commits, one for each issue.

Multiple separate issues require unique pull requests if every pull request is going to be squashed.